snap/file.go

```diff
@@ -27,7 +27,8 @@ import (
 
 // File is the interface to interact with the low-level snap files
 type File interface {
-	Install(targetDir string) error
+	// Install copies the snap file to targetPath (and possibly unpacks it to mountDir)
```

Comment:
I think the corner case is really unpacks (maybe I should just leave it out), it's used by tests, maybe @mvo5 know more